lib/libc/stdlib/malloc/snmalloc/Makefile.inc

```diff
@@ -1,6 +1,8 @@
 .PATH: ${LIBC_SRCTOP}/stdlib/malloc/snmalloc
 
-MISRCS+=	malloc.cc
+MISRCS+=	malloc.c
```

Comment:
Was this change to the file name intentional? It breaks the snmalloc build.

Reply:
Not at all.  I'm not sure how that even made it in there.